src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname: </label>
+        <input 
+        type="text" 
+        id="surname" 
+        name="surname" 
+        required
+        >
```

Comment:
```suggestion
        <input 
          type="text" 
          id="surname" 
          name="surname" 
          required
        >
```

please fix the formatting everywhere (checklist p3)

<img width="992" alt="Screenshot 2023-04-20 at 17 45 44" src="https://user-images.githubusercontent.com/32853034/233418945-1ce6e2b0-5d28-4843-bf8e-99cb4b799287.png">